WordPress/Classes/Stores/StatsInsightsStore.swift

```diff
@@ -947,14 +948,17 @@ extension StatsInsightsStore {
 private extension InsightStoreState {
 
     func storeTodayWidgetData() {
-        guard widgetUsingCurrentSite() else {
-            return
-        }
+
 
         let data = TodayWidgetStats(views: todaysStats?.viewsCount,
                                     visitors: todaysStats?.visitorsCount,
                                     likes: todaysStats?.likesCount,
                                     comments: todaysStats?.commentsCount)
+        saveHomeWidgetTodayData(stats: data)
+
+        guard widgetUsingCurrentSite() else {
```

Comment:
So this is a bit confusing, since I added the method that stores the data for the new widget, inside the old one. 
The `guard` you are looking at is for the old widget. The reason for not separating them was to use the same instance of `TodayWidgetStats` but I realize that this causes confusion, so I'm going to refactor this.